server/src/main/java/org/opensearch/cluster/service/TaskBatcher.java

```diff
@@ -119,26 +137,37 @@ public void submitTasks(List<? extends BatchedTask> tasks, @Nullable TimeValue t
         }
     }
 
-    private void onTimeoutInternal(List<? extends BatchedTask> tasks, TimeValue timeout) {
+    void onTimeoutInternal(List<? extends BatchedTask> tasks, TimeValue timeout) {
```

Comment:
this was marked private earlier

Reply:
I have added new UTs to test that submitting same task after older one was timeout should pass. For that we need to call this function from test cases.

That's why I have removed private and make it as package private.